traitcuration/traits/views.py

```diff
@@ -138,3 +138,8 @@ def zooma_suggestions(request):
 def dummy_data(request):
     dummy.import_dummy_data()
     return redirect('datasources')
+
+
+def ols_queries(request):
+    get_ols_status.delay()
+    return redirect('datasources')
```

Comment:
(I'm adding this here so that we can have a thread rather than isolated comments)

> @tskir testing this functionality is tricky, because it turns out that django-computedfields doesn't play well with the Django admin site. Changing a status value directly from the admin site causes an error in the term. So if you want I could write a script that changes some status values of terms via the Django shell, so that you can test if they are updated correctly when the OLS queries are made.

I think this can be done somewhat easier by just expanding the existing “test suite” of dummy traits which we already have. We could add some dummy traits which have their status purposefully incorrect to start with, and then test the process by clicking the "OLS STATUS UPDATES". Eventually this can also evolve smoothly into an automated test suite, just as the other dummy traits.

Reply:
The admin issue is fixed so feel free to change whatever you like there, however I did add a dummy obsolete term (*/ obsolete_adrenocortical carcinoma / - EFO:0003093*) that is marked as 'current' and a dummy 'current' term (*/ Hereditary breast cancer / - Orphanet:227535*) that is marked as 'awaiting_import' to test the most usual cases right away.